projects/plugins/jetpack/modules/publicize/publicize.php

```diff
@@ -713,15 +729,16 @@ public function get_filtered_connection_data( $selected_post_id = null ) {
 				}
 
 				$connection_list[] = array(
-					'unique_id'     => $unique_id,
-					'service_name'  => $service_name,
-					'service_label' => $this->get_service_label( $service_name ),
-					'display_name'  => $this->get_display_name( $service_name, $connection ),
-
-					'enabled'      => $enabled,
-					'done'         => $done,
-					'toggleable'   => $toggleable,
-					'global'       => 0 == $connection_data['user_id'],
+					'unique_id'       => $unique_id,
+					'service_name'    => $service_name,
+					'service_label'   => $this->get_service_label( $service_name ),
+					'display_name'    => $this->get_display_name( $service_name, $connection ),
+					'profile_picture' => $this->get_profile_picture( $connection ),
+
+					'enabled'         => $enabled,
+					'done'            => $done,
+					'toggleable'      => $toggleable,
+					'global'          => 0 == $connection_data['user_id'],
```

Comment:
I think changing the strictness here might require a follow-up. This same logic is repeated elsewhere in the file.